### PR TITLE
Minimum and maximum LODs can be set from the command line, Clean unus…

### PR DESCRIPTION
**Pull request by bilgili**
…ed classes

### CODE REVIEW

**Comment by bilgili** [comment]
retest this please

**Comment by bilgili** [comment]
retest this please

**Comment by bilgili** [comment]
retest this please

**Comment by bilgili** [comment]
retest this please

**Comment by bilgili** [comment]
retest this please

**Comment by tribal-tec** [comment]
rebase still didn't work? I see eile commit in this PR.

**Comment by tribal-tec** [comment]
Rebase onto master
Good cleanup! +1 after comments are addressed

**Comment by bilgili** [comment]
retest this please

File: livre/Eq/Channel.cpp [hunk 1]
```diff
@@ -295,10 +296,25 @@ class Channel
         livre::Node* node = static_cast< livre::Node* >( _channel->getNode( ));
         livre::Window* window = static_cast< livre::Window* >( _channel->getWindow( ));
         livre::Pipe* pipe = static_cast< livre::Pipe* >( window->getPipe( ));
+
+
+        const uint32_t minDepth = pipe->getFrameData()->getVRParameters()->minDepth;
```
Comment (tribal-tec) [inline livre/Eq/Channel.cpp]: get VRParameters and dashTree as references/shortcuts. They are used multiple times here

File: livre/Lib/Configuration/VolumeRendererParameters.cpp [hunk 2]
```diff
@@ -50,6 +54,10 @@ VolumeRendererParameters::VolumeRendererParameters()
                                    "Screen space error", screenSpaceError );
     configuration_.addDescription( configGroupName_, SYNCHRONOUSMODE_PARAM,
                                    "Enable synchronous mode", synchronousModeEnabled );
+    configuration_.addDescription( configGroupName_, MINDEPTH_PARAM,
+                                   "Sets minimum level of detail", minDepth );
```
Comment (tribal-tec) [inline livre/Lib/Configuration/VolumeRendererParameters.cpp]: Set or no verb at all. Most of the other options have also no verb :)

File: livre/Lib/Render/ScreenSpaceLODEvaluator.h [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+#ifndef _ScreenSpaceLODEvaluator_h_
+#define _ScreenSpaceLODEvaluator_h_
+
+#include <livre/core/Render/LODEvaluator.h>
+
+namespace livre
+{
+
+class ScreenSpaceLODEvaluator : public LODEvaluator
```
Comment (tribal-tec) [inline livre/Lib/Render/ScreenSpaceLODEvaluator.h]: doxygen for class and/or functions?

File: livre/core/Render/LODEvaluator.h [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+/* The base class for selecting a LOD for a given point in view space */
+
+class LODEvaluator
+{
+public:
+    LODEvaluator() {}
+    virtual ~LODEvaluator() {}
+
+    /**
+     * Calcualtes a level of detail with given parameters and the frustum.
```
Comment (tribal-tec) [inline livre/core/Render/LODEvaluator.h]: typo 'Calcualtes'

**Comment by bilgili** [comment]
retest this please

**Comment by tribal-tec** [comment]
+2

**Pull request merged**